Jenkinsfile

```diff
@@ -22,15 +22,15 @@ pipeline {
     stage('Build') {
       steps {
         sh '''
-          eval $(opam config env)
+          eval "$(opam config env)"
```

Comment:
not sure this is actually correct, given that stdout of opam config env may contain whitespace.

Reply:
It does contain whitespace, even in this case. However, the format of the instructions issued makes it so it doesn't matter. Take a look at the perl instructions for `local::lib`, where a similar instruction is used to set one's environment, and they enclose it in quotes: https://metacpan.org/pod/local::libhttps://metacpan.org/pod/local::lib

Reply:
I don't really care what perl does. I care what opam tells you to do, and this is not it.

Reply:
Alright, I'll revert this.